### PR TITLE
Change eslint configuration with prretier option

### PR DESCRIPTION
**Pull request by garnio1987**

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "semi": true,
```
Comment (MadSandwich) [inline .prettierrc.json:2]: This one is actually redundunt I suppose. Only if you have a reason to have it here we can discuss it

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "singleQuote": true,
+  "jsxSingleQuote": false,
+  "bracketSameLine": false,
+  "trailingComma": "all",
+  "printWidth": 100,
+  "tabWidth": 2,
+  "bracketSpacing": true,
+  "arrowParens": "always",
+  "endOfLine": "auto"
```
Comment (MadSandwich) [inline .prettierrc.json:11]: this setting should be set to 'lf', as it set across our others configs

**Review by MadSandwich — APPROVED** [review]

**Comment by MadSandwich** [comment]
Deploy failed as lint can't comply
image

File: .prettierrc.json (hunk 1 above)
Comment (garnio1987) [reply .prettierrc.json:2]: It's a bit hard to be wihout this for me)

**Review by MadSandwich — APPROVED** [review]

**Pull request merged**